dataland-framework-toolbox/inputs/sfdr/sfdr.csv

```diff
@@ -16,97 +16,100 @@ Field Identifier,Category,Sub-Category,Field Name,Tooltip - upload page,Tooltip
+20,Environmental,Greenhouse gas emissions ,Total Revenue,"Total revenue for the financial year. i.e., income arising in the course of an entity's ordinary activities, the amounts derived from the sale of products and the provision of services after deducting sales rebates and value added tax and other taxes directly linked to turnover. Overall turnover is equivalent to a firm's total revenues over some period of time (millions)",,Currency,, ,Extended,,,
+21,Environmental,Greenhouse gas emissions ,Carbon footprint,Tonnes of GHG emissions / million of the enterprise value (in the same currency as the enterprise value), ,Number,,Tonnes / €M Revenue,Extended,,,
+22,Environmental,Greenhouse gas emissions ,GHG intensity,Tonnes of GHG emissions / million of the revenue (in the same currency as the total revenue), ,Number,,Tonnes / €M Revenue,Extended,,,
+23,Environmental,Greenhouse gas emissions ,GHG intensity - scope 1,Tonnes of Scope 1 GHG emissions / million of the revenue (in the same currency as the total revenue),,Number,,Tonnes / €M Revenue,Extended,,,
+24,Environmental,Greenhouse gas emissions ,GHG intensity - scope 2,Tonnes of Scope 2 GHG emissions / million of the revenue (in the same currency as the total revenue),,Number,,Tonnes / €M Revenue,Extended,,,
+25,Environmental,Greenhouse gas emissions ,GHG intensity - scope 3,Tonnes of Scope 3 GHG emissions / million of the revenue (in the same currency as the total revenue),,Number,,Tonnes / €M Revenue,Extended,,,
+26,Environmental,Greenhouse gas emissions ,GHG intensity - scope 4,Tonnes of Scope 4 GHG emissions / million of the revenue (in the same currency as the total revenue),,Number,,Tonnes / €M Revenue,Extended,,,
+27,Environmental,Greenhouse gas emissions ,Fossil Fuel Sector Exposure,"Do you derive any revenues from exploration, mining, extraction, production, processing, storage, refining or distribution, including transportation, storage and trade, of fossil fuels as defined in Article 2, point (62), of Regulation (EU) 2018/1999 of the European Parliament and of the Council? See also Regulation, Annex I, top (5).","(Part of) revenues derived from exploration, mining, extraction, production, processing, storage, refining or distribution, including transportation, storage and trade, of fossil fuels as defined in Article 2, point (62), of Regulation (EU) 2018/1999 of the European Parliament and of the Council? See also Regulation, Annex I, top (5).",Yes/No, ,,Extended,,,
+28,Environmental,Greenhouse gas emissions ,Financed scope 1 and scope 2 emissions,The sum of scope 1 and scope 2 emissions of financed companies,,Number,,Tonnes ,Extended,,,
```

Comment:
Anführungszeichen für den Text "The sum of scope..."

Reply:
s.o.